lib/forwarder.js

```diff
@@ -12,15 +12,26 @@ import { removeConfig } from '../lib/config.js';
  * @param {Config} config
  */
 export async function forwardToDaemon(resolver, config) {
+  const eslintArgs = process.argv.slice();
   const text = process.argv.includes('--stdin') ? await readStdin() : null;
   const { stdout } = supportsColor;
 
+  const fixToStdoutIndex = eslintArgs.indexOf('--fix-to-stdout');
+  const fixToStdout = fixToStdoutIndex !== -1;
+
+  if (fixToStdout) {
+    if (!eslintArgs.includes('--stdin')) {
+      throw new Error('--fix-to-stdout requires passing --stdin as well');
```

Comment:
Ah, you already did it 🙌 – However, it should `console.error` the message, set the exit code to 1 and return. The exception isn't caught anywhere and printing the stack is ugly and not really helping anyone. Thanks.

Reply:
makes sense. Done. Please note the eslint error message that I can't manage to fix. 
Rule: https://eslint.org/docs/latest/rules/require-atomic-updates